uaa/src/test/java/org/cloudfoundry/identity/uaa/orchestrator/RestErrorResponse.java

```diff
@@ -0,0 +1,23 @@
+package org.cloudfoundry.identity.uaa.orchestrator;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+public class RestErrorResponse {
+
```

Comment:
Remove this class and use [OrchestratorZoneResponse](https://github.com/GESoftware-CF/uaa/blob/predix_extensions_75.18.1/zone-service/src/main/java/org/cloudfoundry/identity/uaa/zone/model/OrchestratorZoneResponse.java) instead

Reply:
Done